core/src/com/unciv/logic/battle/Battle.kt

```diff
@@ -242,7 +242,8 @@ object Battle {
                 0.1f + attacker.getAttackingStrength().toFloat() / defender.getDefendingStrength()
                     .toFloat() * 0.4f
             )
-            return Random().nextFloat() <= captureChance
+            /** Between 0 and 1.  Defaults to turn and location-based random to avoid save scumming */
+            return Random(attacker.getCivInfo().gameInfo.turns * defender.getTile().position.hashCode().toLong()).nextFloat() <= captureChance
```

Comment:
```suggestion
            val random = Random(attacker.getCivInfo().gameInfo.turns * defender.getTile().position.hashCode())
            return random.nextFloat() <= captureChance
```
For readability (line is too long) separate 'what is random' and 'what do we do with it'

Reply:
Ok for me, I will try to modify it and run the tests